.gitignore

```diff
@@ -2,3 +2,5 @@
 /dist
 /node_modules
 .pnpm-debug.log
+package-lock.json
```

Comment:
I'd prefer if we didn't `gitignore` this - it's a sign the wrong package manager is being used (`npm` instead of `pnpm`).

Reply:
gotcha, wasn't aware pnpm was a thing, was using npm this whole time